### PR TITLE
get latest wasm filters from latest release

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: ashish <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Assuming not ready for review given the do-not-merge label.

**Comment by leecalcote** [comment]
Please confirm whether this is ready for review or not.

**Comment by Revolyssup** [comment]
> Please confirm whether this is ready for review or not.

Ready

File: Dockerfile [hunk 1]
```diff
@@ -27,10 +27,13 @@ RUN git clone --depth=1 https://github.com/layer5io/wrk2 && cd wrk2 && make
 FROM alpine:3.15 as seed_content
 RUN apk add --no-cache curl
 WORKDIR /
-# bundling filters (temporary hardcoding done below, to be removed after https://github.com/layer5io/wasm-filters/issues/38 is resolved)
-RUN curl -L -s https://github.com/layer5io/wasm-filters/releases/download/v0.1.0/wasm-filters-v0.1.0.tar.gz -o wasm-filters.tar.gz \ 
-    && mkdir -p /seed_content/filters/binaries \
-    && tar xzf wasm-filters.tar.gz --directory=/seed_content/filters/binaries
+RUN lines=$(curl -s https://api.github.com/repos/layer5io/wasm-filters/releases/latest | grep "browser_download_url.*wasm" | cut -d : -f 2,3 | sed 's/"//g') \
```
Comment (leecalcote) [inline Dockerfile:30]: Good deal. The prior series of commands didn't quite work for me when testing.

**Review by leecalcote — APPROVED** [review]

**Pull request merged**